MediaBrowser.Providers/Plugins/Tmdb/TmdbClientManager.cs

```diff
@@ -136,6 +136,56 @@ public async Task<TvShow> GetSeriesAsync(int tmdbId, string language, string ima
             return series;
         }
 
+        /// <summary>
+        /// Gets a tv show episode group from the TMDb API based on the show id and the display order.
+        /// </summary>
+        /// <param name="tvShowId">The tv show's TMDb id.</param>
+        /// <param name="displayOrder">The display order.</param>
+        /// <param name="language">The tv show's language.</param>
+        /// <param name="imageLanguages">A comma-separated list of image languages.</param>
+        /// <param name="cancellationToken">The cancellation token.</param>
+        /// <returns>The TMDb tv show episode group information or null if not found.</returns>
+        private async Task<TvGroupCollection> GetSeriesGroupAsync(int tvShowId, string displayOrder, string language, string imageLanguages, CancellationToken cancellationToken)
+        {
+            var key = $"group-{tvShowId.ToString(CultureInfo.InvariantCulture)}-{displayOrder}-{language}";
+            if (_memoryCache.TryGetValue(key, out TvGroupCollection group))
+            {
+                return group;
+            }
+
+            TvGroupType? groupType =
```

Comment:
This should actually be at the top now that I'm thinking about it. If groupType is null the cache won't have an entry, so the string allocation and the lookup is wasted.

Reply:
Hah, I thought about that for a second, but decided against it (because it moves it farther from where it's used). But yeah I guess it makes sense to move it to the top.